Firestore/core/src/firebase/firestore/model/transform_operation.h

```diff
@@ -148,7 +149,7 @@ class TransformOperation {
   explicit TransformOperation(std::shared_ptr<const Rep> rep);
 
   const Rep& rep() const {
-    return *rep_;
+    return *NOT_NULL(rep_);
```

Comment:
I think we still need to modify all the public methods to call `rep()` instead of `rep_` ?

Reply:
Yes, of course. Doh.

Reply:
[operator<<](https://github.com/firebase/firebase-ios-sdk/blob/wilhuff/mutation-06.5-fixup/Firestore/core/src/firebase/firestore/model/transform_operation.cc#L51) should probably handle rep_ == nullptr as well?

Reply:
I resolved this by making `ToString()` handle `rep_ == nullptr` and then just delegated to that.

```c++
class TransformOperation {
  // ...
  std::string ToString() const {
    return rep_ ? rep().ToString() : "(invalid)";
  }
};

std::ostream& operator<<(std::ostream& os, const TransformOperation& op) {
  return os << op.ToString();
}
```